pmoves/services/agentgym-rl-coordinator/coordinator/training.py

```
@@ -0,0 +1,392 @@
+        resp = await client.post(
+            f"{self.supabase_url}/rest/v1/rpc/create_training_run",
+            headers=self._headers,
+            json={
+                "p_run_id": run_id,
+                "p_config": json.dumps(config) if isinstance(config, dict) else config,
+                "p_total_epochs": total_epochs,
+            },
+        )
+
+        if resp.status_code not in [200, 201]:
+            raise RuntimeError(f"Failed to create training run: {resp.text}")
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)